logrotate.c

```diff
@@ -1511,6 +1512,18 @@ static int findNeedRotating(const struct logInfo *log, unsigned logNum, int forc
                             state->lastRotated.tm_hour, state->lastRotated.tm_min);
                 }
                 break;
+            case ROT_MINUTES:
```

Comment:
You should probably also update the extension logic: https://github.com/logrotate/logrotate/blob/6f06d903d174d019f89647d1259be9d1f2f6102c/logrotate.c#L1885-L1895

Reply:
done